shared/src/business/useCaseHelper/pdf/savePaperServicePdf.test.js

```diff
@@ -0,0 +1,107 @@
+const {
+  applicationContext,
+} = require('../../test/createTestApplicationContext');
+const { savePaperServicePdf } = require('./savePaperServicePdf');
+
+const mockDocumentId = 'cf332ba3-df9e-42c6-b9bc-065da5bb7a36';
+const mockDocumentUrl = 'www.example.com';
+const mockDocument = {
+  getPages: jest.fn(),
+  save: jest.fn(),
+};
+
+describe('savePaperServicePdf', () => {
+  beforeEach(() => {
+    applicationContext.getUniqueId.mockReturnValue(mockDocumentId);
+
+    applicationContext
+      .getPersistenceGateway()
+      .getDownloadPolicyUrl.mockReturnValue({ url: mockDocumentUrl });
+
+    mockDocument.getPages.mockReturnValue({ length: 1 });
+    mockDocument.save.mockReturnValue(mockDocument);
+  });
+
+  it('should not return a url or docketEntryId when there is no paper service on the case', async () => {
```

Comment:
Nit: based on the file above it looks like it does/does not do certain actions based on whether the document passed in has pages. Is that correct? If so, can the test names be updated to reflect that behavior?

Reply:
That is correct! TODO: update test names